src/main/java/at/hannibal2/skyhanni/features/rift/area/dreadfarm/RiftWiltedBerberisHelper.kt

```diff
@@ -216,58 +147,132 @@ object RiftWiltedBerberisHelper {
         if (!hasFarmingToolInHand) return
         if (config.onlyOnFarmland && !isOnFarmland) return
 
-        // original system:
-        if (fallback) {
-            for (berberis in list) {
-                with(berberis) {
-                    if (currentParticles.distanceToPlayer() > 20) continue
-                    if (y == 0.0) continue
-
-                    val location = currentParticles.fixLocation(berberis)
-                    if (!moving) {
-                        event.drawFilledBoundingBoxNea(axisAlignedBB(location), Color.YELLOW, 0.7f)
-                        event.drawDynamicText(location.up(), "§eWilted Berberis", 1.5, ignoreBlocks = false)
-                    } else {
-                        event.drawFilledBoundingBoxNea(axisAlignedBB(location), Color.WHITE, 0.5f)
-                        previous?.fixLocation(berberis)?.let {
-                            event.drawFilledBoundingBoxNea(axisAlignedBB(it), Color.LIGHT_GRAY, 0.2f)
-                            event.draw3DLine(it.add(0.5, 0.0, 0.5), location.add(0.5, 0.0, 0.5), Color.WHITE, 3, false)
-                        }
-                    }
+        if (fallback) fallbackRender(event)
+        else primaryRender(event)
+    }
+
+    @HandleEvent
+    fun onConfigFix(event: ConfigUpdaterMigrator.ConfigFixEvent) {
+        event.move(60, "rift.area.dreadfarm.wiltedBerberis.hideparticles", "rift.area.dreadfarm.wiltedBerberis.hideParticles")
```

Comment:
You need to bump the config version number to 71 (at this moment), or the migration won’t work. Don’t forget to also update the config version in the [ConfigUpdaterMigrator](https://github.com/hannibal002/SkyHanni/blob/2fe67fcf9be36f6718219916dc2d52c9ee3d73d5/src/main/java/at/hannibal2/skyhanni/config/ConfigUpdaterMigrator.kt#L15).